cst438_gradebook/src/main/java/com/cst438/domain/Assignment.java

```diff
@@ -25,7 +25,7 @@ public class Assignment {
 	@OneToMany(mappedBy="assignment")
 	private List<AssignmentGrade> assignmentGrades;
 	
-	private String name;
+	public String name;
```

Comment:
Maybe you can just use a setter to set a name. Converting this property to public does not feel right.